packages/open-next/src/core/requestHandler.ts

```diff
@@ -19,7 +19,9 @@ import { requestHandler, setNextjsPrebundledReact } from "./util";
 // This is used to identify requests in the cache
 globalThis.__openNextAls = new AsyncLocalStorage();
 
+//#override patchAsyncStorage
```

Comment:
Long term, we should not delete it but have a cloudflare specific impl.
Maybe we could then override instead of deleting but at the same time the temporary patch in cloudflare is good enough before we have a long term solution.

WDYT?

Reply:
This one will hopefully be removed in the future in OpenNext as well.
Once this PR https://github.com/vercel/next.js/pull/72082 gets merged it will not be necessary anymore.
I just have to figure out how to test with the new `use cache` stuff from next